implementations/rust/ockam/ockam_command/src/forwarder/create.rs

```diff
@@ -108,13 +90,11 @@ async fn rpc(ctx: Context, (opts, cmd): (CommandGlobalOpts, CreateCommand)) -> R
         } else {
             cmd.forwarder_name.clone()
         };
-        let body = CreateForwarder::new(
-            ma,
-            Some(alias),
-            at_rust_node,
-            pa,
-            CredentialExchangeMode::Oneway,
-        );
+        let body = if Some(Project::CODE) == cmd.at.first().map(|p| p.code()) {
+            CreateForwarder::at_project(ma, Some(alias), cmd.cloud_opts.route())
+        } else {
+            CreateForwarder::at_node(ma, Some(alias), at_rust_node, cmd.authorized)
```

Comment:
Should we error if `cmd.cloud_opts` was provided as well?

Reply:
AFAIK they read from an environment variable and default to a standard address if not present. `CloudOpts` has no arguments to provide.